test/blockchain.cpp

```diff
@@ -47,6 +51,7 @@ void doBlockchainTests(json_spirit::mValue& _v, bool _fillin)
 		cerr << i.first << endl;
 		mObject& o = i.second.get_obj();
 
+		vector<BlockInfo> vBiBlocks;
```

Comment:
you only need vBiBlocks in the _fillin == true branch, so you should move it there. Keep the valid scope of a variable as small as possible.